### DISCUSSION TITLE
Updates for 2023-2024

### DISCUSSION THREAD

**Discussion by brunoalbertopeixoto** [AppFlowy-IO/AppFlowy#3767] [Ideas]
I simply love this app. I imagine it will be difficult to program, but I have a series of suggestions and development for this application. I joined it when I was looking for an alternative to Trello and Notion. Suggestions:
1. Android version.
2. Auto serve host without programming. To understand my suggestion, I think the application could synchronize like Synthing does or using tunneling like HopToDesk.
3. Merged organization method like the Priority Matrix Eisenhower App for Android ( https://appfluence.com/eisenhower-matrix-app/ https://play.google.com/store/apps/details?id=com.soffective.lifebalance

**Comment by annieappflowy** [comment]
> I simply love this app. I imagine it will be difficult to program, but I have a series of suggestions and development for this application. I joined it when I was looking for an alternative to Trello and Notion. Suggestions:
> 
> 1. Android version.
> 2. Auto serve host without programming. To understand my suggestion, I think the application could synchronize like Synthing does or using tunneling like HopToDesk.
> 3. Merged organization method like the Priority Matrix Eisenhower App for Android ( https://appfluence.com/eisenhower-matrix-app/ https://play.google.com/store/apps/details?id=com.soffective.lifebalance

Update: you can download AppFlowy for Android from our website ([link](https://appflowy.io/download#ios-and-android))
